packages/leveldb/src/leveldb/internal.ts

```diff
@@ -1,13 +1,17 @@
 import { RxDocumentData, RxJsonSchema, getPrimaryFieldOfPrimaryKey } from "rxdb";
-import { Level } from 'level';
+import Level from 'level';
+import pull from 'pull-stream';
+import pullLevel from 'pull-level';
```

Comment:
`pull-stream` is an elegant async-iterator library that Scuttlebutt uses extensively.